src/django/api/management/commands/notify.py

```diff
@@ -1,16 +1,88 @@
-        group = parser.add_argument_group('required arguments')
-        group.add_argument('-e', '--event-id',
-                           required=True,
-                           help='The ID of an existingEvent record.')
+        group = parser.add_argument_group("required arguments")
+        group.add_argument(
+            "-e",
+            "--event-id",
+            required=True,
+            help="The ID of an existingEvent record."
+        )
 
     def handle(self, *args, **options):
-        print('TODO: REPLACE THIS STUB IMPLEMENTATION')
+        notification_webhook_timeout = getattr(
+            settings, "NOTIFICATION_WEBHOOK_TIMEOUT", 10
+        )
+        ROLLBAR = getattr(settings, "ROLLBAR", {})
+
+        event_id = options["event_id"]
+        event = Event.objects.get(pk=event_id)
+
+        cwh = ContributorWebhook.objects.order_by("created_at")
+
+        for contributor_webhook in cwh:
+            try:
+                query_params = QueryDict(
+                    contributor_webhook.filter_query_string.replace("?", ""),
+                    mutable=True,
+                )
+                query_params.update({"id": event.object_id})
+
+                if (
+                    contributor_webhook.notification_type
+                    == ContributorWebhook.ASSOCIATED
+                ):
+                    query_params.update(
+                        {"contributors": contributor_webhook.contributor_id}
+                    )
+
+                facilities_qs = Facility \
+                    .objects \
+                    .filter_by_query_params(query_params)
+
+                if facilities_qs.count() > 0:
+                    body = {
+                        "event_type": event.event_type,
+                        "event_time": event.event_time.isoformat(),
+                        "event_details": event.event_details,
+                    }
+                    response = requests.post(
+                        contributor_webhook.url,
+                        json=body,
+                        timeout=notification_webhook_timeout,
+                    )
+                    status = (
+                        ContributorWebhook.DELIVERED
+                        if (response.status_code >= 200
+                            and response.status_code < 300)
+                        else ContributorWebhook.FAILED
+                    )
+                else:
+                    status = ContributorWebhook.SKIPPED
+            except:  # noqa: E722
+                status = ContributorWebhook.FAILED
+
+                if ROLLBAR:
+                    import rollbar
+
+                    rollbar.report_exc_info(
+                        sys.exc_info(),
+                        extra_data={
+                            "contributor_webhook_id": contributor_webhook.id,
+                        },
+                    )
+                else:
+                    self.stderr.write(sys.exc_info())
```

Comment:
To test this I made my webhook URL invalid `https://testtesttesttest.free.beeceptor.commm`. When I ran the notify command I got the following exception

```
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/local/src/api/management/commands/notify.py", line 86, in handle
    self.stderr.write(sys.exc_info())
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 152, in write
    if ending and not msg.endswith(ending):
AttributeError: 'tuple' object has no attribute 'endswith'
```


Here is a propose solution that replaced stderr with the logger

```diff
diff --git a/src/django/api/management/commands/notify.py b/src/django/api/management/commands/notify.py
index a31a4e87..6b2f7075 100644
--- a/src/django/api/management/commands/notify.py
+++ b/src/django/api/management/commands/notify.py
@@ -1,3 +1,4 @@
+import logging
 import requests
 from django.http import QueryDict
 from django.conf import settings
@@ -6,6 +7,9 @@ from django.core.management.base import BaseCommand
 from api.models import ContributorWebhook, Event, Facility
 
 
+logger = logging.getLogger(__name__)
+
+
 class Command(BaseCommand):
     help = "POST notifications for an Event to configured webhook endpoints"
 
@@ -83,6 +87,8 @@ class Command(BaseCommand):
                         },
                     )
                 else:
-                    self.stderr.write(sys.exc_info())
+                    logger.exception(
+                        "Exception delivering event {} to webhook {}".format(
+                            event_id, contributor_webhook.id))
 
             contributor_webhook.log_event(event, status)
```